config/initializers/job_configurations.rb

```diff
@@ -204,5 +204,5 @@
   name: 'Partner API report',
   interval: 24 * 60 * 60, # 24 hours
   timeout: 300,
-  callback: -> { Agreements::Reports::PartnerApiReport.new.call },
+  callback: -> { Agreements::Reports::PartnerApiReport.new.run },
```

Comment:
this isn't the first time we've had typos in this file.... WDYT of switching it to be something that would be more aggressive finding typos? Like the entire file

```suggestion
  callback: Agreements::Reports::PartnerApiReport.new.method(:run)
```

Reply:
This is great, but unfortunately creates some autoload warnings since it refers to classes that haven't been loaded yet:

```
DEPRECATION WARNING: Initialization autoloaded the constants Agreements, Agreements::Reports, and Agreements::Reports::PartnerApiReport
```

Reverting to original :/

Reply:
😭

Reply:
😭